api/handlers.go

```diff
@@ -676,12 +695,15 @@ func (si *ServerImplementation) SearchForApplicationBoxes(ctx echo.Context, appl
 	}
 	happyResponse := generated.BoxesResponse{ApplicationId: applicationID, Boxes: []generated.BoxDescriptor{}}
 
+	requestedLimit := uint64(20) // Default limit for boxes
```

Comment:
Why is this done differently?  The others had some structure that contained the limit?